src/pages/team.js

```diff
@@ -0,0 +1,97 @@
+import Head from "next/head";
+
+const people = [
+  {
+    name: 'Vincent Villafuerte',
+    role: 'WebX Core',
+    imageUrl:
+      'https://github.com/vinzvinci.png',
+    githubUrl: 'https://github.com/vinzvinci',
+    twitterUrl: 'https://twitter.com/vinzvinci',
+    linkedinUrl: 'https://www.linkedin.com/in/vinzvinci/',
+  },
+  {
+    name: 'Shubham Kukreti',
+    role: 'WebX Core',
+    imageUrl:
+      'https://github.com/KukretiShubham.png',
+    githubUrl: 'https://github.com/KukretiShubham',
+    twitterUrl: 'https://twitter.com/ShubhamKukretii',
+    linkedinUrl: 'https://www.linkedin.com/in/shubhamkukreti/',
+  },
+  {
+    name: 'Ritvik Shukla',
+    role: 'WebX Core',
+    imageUrl:
+      'https://github.com/shuklaritvik06.png',
+    githubUrl: 'https://github.com/shuklaritvik06',
+    twitterUrl: 'https://twitter.com/shuklaritvik06',
+    linkedinUrl: 'https://www.linkedin.com/in/ritvikshukla/',
+  },
+  {
+    name: 'Maxime Kubik',
+    role: 'WebX Core',
+    imageUrl:
+      'https://github.com/mkubdev.png',
+    githubUrl: 'https://github.com/mkubdev',
+    twitterUrl: 'https://twitter.com/mkubdev',
+    linkedinUrl: 'https://www.linkedin.com/in/maximekbk/',
```

Comment:
Question: shouldn't we add the other maintainers as well, or only WebX Core counts? 🤔

Reply:
I'm thinking to add them below the WebX Core later using grid with round images.

Reply:
> Question: shouldn't we add the other maintainers as well, or only WebX Core counts? thinking

For us everyone matters equally David! Any Org is by its members then its founders :) I agree with  vinci's thoughts